source/extensions/filters/http/aws_lambda/BUILD

```diff
@@ -76,8 +76,7 @@ envoy_cc_library(
     repository = "@envoy",
     deps = [
         ":aws_lambda_filter_lib",
-        "@envoy//include/envoy/registry",
```

Comment:
Am I correct in understanding that we don't need the `registry` dep anymore which is why it's removed here?

Reply:
we might need it, i removed it because i thought the path changed; it turned out that it didnt. i think it still works because it is included transitivly